x-pack/plugins/fleet/server/services/artifacts/artifacts.test.ts

```diff
@@ -186,13 +186,25 @@ describe('When using the artifacts services', () => {
     });
 
     it('should create and return a multiple big artifacts', async () => {
-      const { ...generatedArtifact1 } = generateArtifactMock({ encodedSize: 5_000_500 });
+      const { ...generatedArtifact1 } = generateArtifactMock({
+        encodedSize: 5_000_500,
+        decodedSha256: '1234',
+      });
       const newBigArtifact1 = generatedArtifact1;
```

Comment:
sorry for the rookie question, but what is the reason to save the reference to a new const?